astropy/io/fits/tests/test_image.py

```diff
@@ -1959,3 +1959,17 @@ def test_image_write_readonly(tmpdir):
 
     with fits.open(filename) as hdulist:
         assert_equal(hdulist[1].data, [1.0, 2.0, 3.0])
+
+
+def test_int8(tmp_path):
+    '''Test for int8 support, https://github.com/astropy/astropy/issues/11995'''
+    img = np.arange(-50, 50).astype(dtype=np.int8).reshape(10, 10)
```

Comment:
can just be `np.arange(-50, 50, dtype=np.int8)`

Reply:
done